annotation-file-utilities/src/annotator/find/ASTPathCriterion.java

```diff
@@ -140,7 +135,7 @@ public boolean isSatisfiedBy(TreePath path) {
 
         int astPathLen = astPath.size();
         int actualPathLen = actualPath.size();
-        if (astPathLen == 0 || actualPathLen == 0) { return false; }
+        if (actualPathLen <= 0) { return false; }
```

Comment:
How can actualPathLen, which is actualPath.size() where actualPath is a List, be negative?  This looks like a wrong test to me.